api-testing/multicloud-api/.gitignore

```diff
@@ -0,0 +1,11 @@
+*.iml
```

Comment:
Please remove this file if not required

Reply:
If [PR-2](https://github.com/sodafoundation/qa/pull/2) will be merge automatically this changes remove.